File: 3st/Jaeyoung/최댓값과 최솟값.cs
using System;
public class Solution {
    public string solution(string s) {
        string[] num = s.Split();
        int[] n = new int[num.Length];
        for ( int i = 0 ; i < num.Length ; i++ ) {
            n[i] = int.Parse(num[i]);
        }
        Array.Sort(n);

        return $"{n[0]} {n[n.Length-1]}";
    }
}

File: 3st/Hajeong/programmer/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] a = s.split(" ");
        int[] num = new int[a.length];
        for (int i = 0; i < a.length; i++) {
            num[i] = Integer.parseInt(a[i]);
        }

        Arrays.sort(num);

        answer += num[0] + " " + num[num.length - 1];
        return answer;
    }
}